integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/session/SearchSessionWritePlanBaseIT.java

```diff
@@ -210,6 +210,87 @@ public void earlyProcess() {
 		backend1Mock.verifyExpectationsMet();
 	}
 
+	@Test
+	public void sessionClear() {
```

Comment:
Since you're not using the `SearchSessionWritePlan` API, I think this test is about automatic indexing, and should be moved to `AutomaticIndexingBasicIT`, or a new test class in the test package if you deem it necessary.